src/App.vue

```diff
@@ -430,7 +433,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // open confirmation dialog and wait for response
     this.$refs.confirm.open(
       'Unsaved Changes',
-      'You have unsaved changes in your Incorporation Application. Do you want to exit?',
+      'This will cancel your alteration filing. Do you want to exit?',
```

Comment:
Does this code also apply to corrections?

Reply:
This dialog does not.